README.md

```diff
@@ -49,7 +49,7 @@ The Semgrep ecosystem includes:
 
 - Semgrep - the open-source command line tool at the heart of everything (this project)
 - [Semgrep CI](https://semgrep.dev/docs/semgrep-ci/) - a specialized Docker image for running Semgrep in CI environments
-- [Semgrep Playground](https://semgrep.dev/editor) - an online interactive editor for writing and sharing rules
+- [Semgrep Playground](https://semgrep.dev/editor) - an online interactive tool for writing and sharing rules
```

Comment:
Why this change? Editor is more descriptive than tool, even if it's no longer the primary name